src/extension.ts

```diff
@@ -172,14 +185,24 @@ class RunLineInEditor {
             .then(target => replaceContent(target, JSON.stringify({ 'Running command': line }) + '\n')
                 .then(() => exec(line))
                 .then(({ stdout }) => stdout, ({ stdout, stderr }) => JSON.stringify({ stderr, stdout }, null, '    '))
-                .then(content => replaceContent(target, content)
-                    .then(() => this.parsedResult = JSON.parse(content))
-                    .then(undefined, err => {})
-                )
+                .then(content => {
+                    replaceContent(target, content)
```

Comment:
`return` to keep the promise chaining? Or move the two lines below to `hideStatusBarItem()` and keep this unchanged.

Reply:
Fixed